developerportal/templates/molecules/search-result.html

```diff
@@ -0,0 +1,39 @@
+{% load wagtailcore_tags %}
+{% load app_filters %}
+
+<li class="mzp-c-card mzp-c-card-medium search-result-item">
+  <a
+    href="{% pageurl page %}"
+    class="mzp-c-card-block-link
+    {% if page.resource_type == "video" and page.is_external %}
+      js-modal-trigger{% endif %}"
+    {% if page.is_external %}
+      target="_blank"
+      rel="nofollow noopener"
+      data-title="{{ page.title }}"
+      data-type="{{ page.resource_type }}"
+    {% endif %}
+  >
+  <h4>
```

Comment:
I notice that we jump from an `h1` element to an `h4`. Not a super big deal I guess, it would be better if these were `h2` elements though. The styling can remain the same in terms of font size

Reply:
Done - cheers!